src/utils/bookingUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { ISeat, ISeatStatus } from "../types";
+
+interface Ipdetails {
```

Comment:
Any reason to use Ipdetails as generally the format used is IPdetails. If this interface is meant for passenger details, i believe IPassengerDetails might be more understandable